tasks/lib/args.js

```diff
@@ -20,6 +20,13 @@ const args = yargs
     describe: 'Log additional data'
   })
 
+  .option('vendor', {
+    string: true,
+    default: 'chrome',
+    describe: 'Compile the extension for different vendors',
+    choices: ['chrome', 'moz', 'opera']
```

Comment:
I changed from the original `firefox` key to `moz` so that we can reuse this value to generate a vendor-specific extension URL (`moz-extension://` on Firefox, `chrome-extension://` on Chrome and `opera-extension://` on Opera).